Source/UI/NewUI/PaymentForm/Factory/Theme/BillingForm/Theme+BillingCountryInput.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Theme+BillingCountryInput.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 10/08/2022.
+//
+
+import UIKit
+
+public extension Theme {
+    
+    /// Theme generated Billing Country Input Style
+    struct ThemeBillingCountryInput: CellButtonStyle {
+        public var isMandatory: Bool = true
+        public var backgroundColor: UIColor = .clear
+        public var button: ElementButtonStyle
+        public var title: ElementStyle?
+        public var hint: ElementStyle?
+        public var mandatory: ElementStyle?
+        public var error: ElementErrorViewStyle?
+    }
+    
+    /// Create a Billing Country Input from provided themed components
+    func buildBillingCountryInput(button: CountryListButton,
+                                  title: ThemeTitle,
+                                  mandatory: ThemeMandatory? = nil,
+                                  subtitle: ThemeSubtitle? = nil,
+                                  error: ThemeError? = nil) -> ThemeBillingCountryInput {
+        ThemeBillingCountryInput(button: button,
+                                 title: title,
+                                 hint: subtitle,
+                                 mandatory: mandatory,
+                                 error: error)
+    }
+    
+    /// Create a Billing Country Input from provided content
+    func buildBillingCountryInput(buttonText: String,
+                                  buttonImage: UIImage? = nil,
+                                  title: String,
+                                  subtitle: String? = nil,
+                                  subtitleImage: UIImage? = nil,
+                                  isRequiredText: String? = nil,
+                                  errorText: String? = nil,
+                                  errorImage: UIImage? = nil) -> ThemeBillingCountryInput {
+        let subtitleText = subtitle ?? ""
+        let showSubtitle = !subtitleText.isEmpty || subtitleImage != nil
+        
+        let errorText = errorText ?? ""
+        let showError = !errorText.isEmpty || errorImage != nil
+        
+        let mandatoryText = isRequiredText ?? ""
+        let showMandatory = !mandatoryText.isEmpty
+        
+        let button = buildCountryListButton(text: buttonText, image: buttonImage)
```

Comment:
Question : @alex-ioja-yang-cko do you think we can remove build prefix here and throughout the PR ?. 

My suggestion was to just have method name for example `countryListButton, billingHeader` `billingInput` please correct me if I have missed something.

Reply:
Thanks, I was looking at the `Omit needless words. Every word in a name should convey salient information at the use site.` from https://www.swift.org/documentation/api-design-guidelines/ .Is my understanding correct to tell we can then use `build(_ countryListButton text: buttonText, image: buttonImage)` ?

Reply:
@deepesh-vasthimal-cko I am not sure I fully understand that code suggestion. Would you be able to write it as code? 

Like, we now have:
```Swift
func buildCountryListButton(text: String, image: UIImage?)
```
which at consumer use case will become:
```Swift
theme.buildCountryListButton(text: myText, image: myImage)
```

How would your suggestion look?

Reply:
You are right, I got confused with parameter prefix with function naming itself. In this case I was trying to get rid of the redundant word `build` but it we have to satisfy the verb then it makes sense.

Reply:
My goal is to make the use site as easy to read and set expectations as possible. So to be fair I think vast majority of time public methods should start with a verb as it will just naturally improve readability:
object.**do**Something()